cmd/finch/nerdctl_test.go

```diff
@@ -80,14 +82,14 @@ func TestNerdctlCommand_run(t *testing.T) {
 		cmdName string
 		args    []string
 		wantErr error
-		mockSvc func(*mocks.LimaCmdCreator, *mocks.Logger, *gomock.Controller)
+		mockSvc func(*mocks.LimaCmdCreator, *mocks.NerdctlCommandSystemDeps, *mocks.Logger, *gomock.Controller)
 	}{
 		{
 			name:    "happy path",
 			cmdName: "build",
 			args:    []string{"-t", "demo", "."},
 			wantErr: nil,
-			mockSvc: func(lcc *mocks.LimaCmdCreator, logger *mocks.Logger, ctrl *gomock.Controller) {
+			mockSvc: func(lcc *mocks.LimaCmdCreator, nsd *mocks.NerdctlCommandSystemDeps, logger *mocks.Logger, ctrl *gomock.Controller) {
```

Comment:
nit: maybe ncsd to make it consistent with its full name

```suggestion
			mockSvc: func(lcc *mocks.LimaCmdCreator, ncsd *mocks.NerdctlCommandSystemDeps, logger *mocks.Logger, ctrl *gomock.Controller) {
```

Reply:
Updated.